pip-dep/requirements_dev.txt

```diff
@@ -7,6 +7,7 @@ pre-commit>=1.16.1
 pyopenssl>=19.0.0
 pyshark>=0.4.2.4
 pytest>=4.5.0
+scipy
```

Comment:
Do we need it?

Reply:
I think we do not need it since ```travis``` did not complain without it.